components/style/values/specified/mod.rs

```diff
@@ -58,6 +58,8 @@ pub use self::percentage::Percentage;
 pub use self::position::{Position, PositionComponent, GridAutoFlow, GridTemplateAreas};
 pub use self::svg::{SVGLength, SVGOpacity, SVGPaint, SVGPaintKind};
 pub use self::svg::{SVGPaintOrder, SVGStrokeDashArray, SVGWidth};
+#[cfg(feature = "gecko")]
```

Comment:
I'm still not sure why the `cfg` checks are needed, `values/specified/svg` doesn't look Gecko-specific.

Does it build fine if you remove all of them?

Reply:
Yes, it does.